css/ampshared.css

```
@@ -539,6 +539,26 @@ amp-accordion {
   display: block !important;
 }
 
+/* 
+This media query is used instead of @supports due to low compatibility.
+At the time of this change (4/6/2021) no version of Webkit support @supports queries.
+
+These provide low-specificity UI styles to prevent CLS for the header and content 
+segments of accordions.
+*/
```

Nit to make multi-line comment more consistent with other such comments in this file (which aren't all consistent either):

```suggestion
/**
 * This media query is used instead of @supports due to low compatibility.
 * At the time of this change (4/6/2021) no version of Webkit support @supports queries.
 *
 * These provide low-specificity UI styles to prevent CLS for the header and content 
segments of accordions.
 */
```

---

Very good point, merging your suggestion.